app/src/test/java/org/simple/clinic/settings/SettingsInitTest.kt

```diff
@@ -13,17 +13,17 @@ class SettingsInitTest {
   private val spec = InitSpec<SettingsModel, SettingsEffect>(SettingsInit())
 
   @Test
-  fun `when screen is created, then user details, current language & app version should be loaded`() {
+  fun `when screen is created, then user details, current language & app version should be loaded & check for app update availability`() {
```

Comment:
I think we can leave the implementation details up to the method since the `hasEffects` matcher already tells us what is happening, I think we can just rename this to `when screen is created, then the initial data must be loaded`.

Reply:
Same comment for the other method where we test restoring of the model.